request.d.ts

```diff
@@ -0,0 +1,6 @@
+import { PinejsClientCore } from './core'
+import * as request from 'request'
```

Comment:
Ah, interesting! Generally, if you want to debug this kind of thing, you want `tsc --traceResolution`, which will print out every single module searched for, everwhere TS looked for it, what it ended up resolving to and why.

In this case, if you do that it says that it's doing 'classic' module resolution (as opposed to 'node'), which is not what you want. That resolves weirdly, and doesn't start in `node_modules`. I know this is the default for AMD compilation, I'd guess it's doing it here because you're using UMD. If you set `"moduleResolution": "node"` in tsconfig.json it'll do what you expect, and everything'll be fine (although you probably do also want to depend on `@types/request`, if you want to be able to import that)